Fix bloat-gathering script

SammyVimes opened the pull request:
### Changelog entry <!-- a user-readable short description of the changes that goes to CHANGELOG.md and Release Notes -->
Chrome trace format has changed due to clang version update, this PR changes the handling of the traces

### Changelog category <!-- remove all except one -->
* Not for changelog (changelog entry is not required)

SammyVimes committed:
Fix bloat-gathering script

SammyVimes committed:
Temporally comment template bloat

maximyurchuk commented on .github/actions/build_analytics/action.yml:
```diff
@@ -33,7 +33,7 @@ runs:
         export build_target="${{ inputs.build_target }}" 
         ./ya tool bloat --linker-map ydb/apps/ydbd/ydbd.map.lld --input ydb/apps/ydbd/ydbd --save-html ya_bloat_html --save-json bloat.json
-        ./ydb/ci/build_bloat/template_bloat.py -j bloat.json -o template_bloat -t template_bloat_html
+        # ./ydb/ci/build_bloat/template_bloat.py -j bloat.json -o template_bloat -t template_bloat_html
```
А можешь плз написать почему закомменчено? Комментарием, а то не вспомним же

SammyVimes committed:
Disable bloat.json parsing and template_bloat syncing

Commented out lines related to parsing bloat.json and syncing template_bloat files until further development.

SammyVimes committed:
.
